spec/_attachments/requests.cddl

```diff
@@ -73,8 +75,39 @@ query-response = tagged<{
   ? error_code: text
 }>
 
-call-reply = {
+; A request as submitted to /api/v2/.../signed_query
+signed-query-request = envelope<signed-query-content>
+signed-query-content = {
+  request_type: "query"
```

Comment:
I believe the request type needs to be signed_query as well

Reply:
Then it'd be more difficult to merge `/api/v2/.../signed_query` with `/api/v2/.../query` as a URL redirect would not be sufficient (clients would also need to adapt their CBOR payloads).

Reply:
This conversation has been resolved offline.